DEFCORE.md

```diff
@@ -0,0 +1,101 @@
+# Running RefStack against RPC-OpenStack
+
+The [RefStack](https://refstack.openstack.org/#/) tool runs a specific set of
+Tempest tests against an OpenStack deployment to ensure it meets the
+interoperability requirements for OpenStack clouds. This is often called
+"DefCore".
+
+Before getting started, ensure that you have the following:
+
+* A **fully deployed RPC environment**.
+* **AIO RPC deployments are fine**, so long as they have spare RAM/disk
```

Comment:
imho this is a bracketed addendum to the first point, rather than it's own standalone point